docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -589,6 +595,13 @@ class DebugPage extends React.PureComponent<Props, State> {
       debugResponse,
     });
   }
+
+  private moveScroll() {
```

Comment:
How about renaming to `scrollToDebugForm()` to make it more meaningful?

Reply:
I'll renaming method! Thank you for review!